.github/scripts/gpu-opts.sh

```
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Sets $gpu_opts from $job_device and $job_interface.
+# Usage: source .github/scripts/gpu-opts.sh
+
+gpu_opts=""
+if [ "$job_device" = "gpu" ]; then
```

Shellcheck (0.11.0)

[warning] job_device is referenced but not assigned.

(SC2154)